Python/Twitter_Hashtags/README.md

```diff
@@ -0,0 +1,37 @@
+# Tweet scraper using a particular hashtag without the intervention of twitter API
+
+- Here, we make use of snscrape to scrape tweets associated with a particular hashtag. Snscrape is a python library that scrapes twitter without the use of API keys.
+
+- In this script, we take the input of a hashtag and generate details associated with it (Eg., id, username, date, time, the tweet content, number of replies, likes, retweets, quotes and also its URL) tracing upto the desired number of tweets. 
+
+- We store all this information in a .csv file with the selected file name.
+
+## Setup Instructions
+
+### Requirements
+
+1. snscript library: (Supported for Python 3.8 and above only) It's development version is installed as
+
+```
+pip install git+https://github.com/JustAnotherArchivist/snscrape.git
```

Comment:
Is it available on PyPi? We don't need to install git repos and then build it from scratch